src/app/public/modules/datepicker/datepicker.component.spec.ts

```diff
@@ -208,14 +242,16 @@ describe('datepicker', () => {
     let component: DatepickerTestComponent;
     let nativeElement: HTMLElement;
 
-    beforeEach(() => {
+    beforeEach(fakeAsync(() => {
```

Comment:
To help explain this to future devs, would you mind a comment on why the `fakeAsync` is necessary here?